### ISSUE TITLE
Encryption key owners

### ISSUE THREAD

**Issue by ChasNelson1990** [fjelltopp/zarr-ckan#106]
Secret encryption keys will be kept safe. Accessibility to the keys will be controlled and audited. A minimum of 2 and maximum of 3 authorised users should have access to the secret encryption keys.

Relates #1

INGO ME to explore who these individuals will be.
